lib/timeline/component/item/BackgroundItem.js

```diff
@@ -105,16 +99,73 @@ BackgroundItem.prototype.redraw = function() {
     // update class
     var className = (this.data.className ? (' ' + this.data.className) : '') +
         (this.selected ? ' vis-selected' : '');
-    dom.box.className = this.baseClassName + className;
+    this.dom.box.className = this.baseClassName + className;
+  }
+}
+BackgroundItem.prototype._getDomComponentsSizes = function() {
+  // determine from css whether this box has overflow
+  this.overflow = window.getComputedStyle(this.dom.content).overflow !== 'hidden';
+  return {
+    content: {
+      width: this.dom.content.offsetWidth
+    }
+  }
+}
+BackgroundItem.prototype._updateDomComponentsSizes = function(sizes) {
+  // recalculate size
+  this.props.content.width = sizes.content.width;
+  this.height = 0; // set height zero, so this item will be ignored when stacking items
 
-    // determine from css whether this box has overflow
-    this.overflow = window.getComputedStyle(dom.content).overflow !== 'hidden';
+  this.dirty = false;
+}
 
-    // recalculate size
-    this.props.content.width = this.dom.content.offsetWidth;
-    this.height = 0; // set height zero, so this item will be ignored when stacking items
+BackgroundItem.prototype._repaintDomAdditionals = function() {
```

Comment:
Why is the empty method needed?

Reply:
because all the other items contain 6 actions, I wanted to insure that all items have the same number of actions (and that the same function type is done in parallel - this is a preparation to making the redraw queue for all the items defined in the `Item.js` prototype) + it's a space keeper for future implementations of dom additionals to background items (I've been planning to add)